.github/workflows/release-azure-webapp-v2.yaml

```diff
@@ -0,0 +1,258 @@
+on:
+  workflow_call:
+    inputs:
+      workspace_name:
+        description: The name of the workspace to create the artifact for.
+        type: string
+        required: true
+      environment:
+        description: Environment where the artifact will be deployed.
+        type: string
+        required: true
+      resource_group_name:
+        description: Web App resource group name.
+        type: string
+        required: true
+      web_app_name:
+        description: Web App name.
+        type: string
+        required: true
+      use_staging_slot:
```

Comment:
question: in which cases we don't want to deploy to staging slot?

Reply:
When it doesn't exist, think also to lower environments than production